src/linkml_transformer/datamodel/transformer_model.yaml

```diff
@@ -40,6 +40,9 @@ classes:
         description: human readable title for this transformation specification
       prefixes:
         description: maps prefixes to URL expansions
+        range: KeyVal
```

Comment:
Not specific to this line of the file, but can/should there be a specification of the language used for `expr`? I know it's supposed to be in the language agnostic linkml expression syntax (*cough* python *cough*), but it would be good to explicitly specify this, so that it is then possible to use other languages for expressing these transformations.

Reply:
We should definitely do this! this is now a little easier as we have https://github.com/linkml/semantic-dsl